src/Microsoft.ML.StandardLearners/Microsoft.ML.StandardLearners.csproj

```diff
@@ -1,11 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <IncludeInPackage>Microsoft.ML</IncludeInPackage>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="MathNet.Numerics.Signed" Version="$(MathNumericPackageVersion)" />
```

Comment:
note: you'll need to add this PackageReference to our NuGet package as well: 

[https://github.com/dotnet/machinelearning/blob/f222025252f6a2bb6fe6dd79cb80bc627273c58f/pkg/Microsoft.ML/Microsoft.ML.nupkgproj#L11-L17](https://github.com/dotnet/machinelearning/blob/f222025252f6a2bb6fe6dd79cb80bc627273c58f/pkg/Microsoft.ML/Microsoft.ML.nupkgproj#L11-L17)